src/RetryRestClient.js

```diff
@@ -81,15 +81,18 @@ RetryRestClient.prototype.handleRetry = function(method, args) {
     return this.restClient[method].apply(this.restClient, args);
   }
 
+  // The formula used to calculate the individual timeouts is:
+  // (1 + Math.random()) * minTimeout * Math.pow(factor, attempt)
```

Comment:
Might be easier to link to their docs:
https://github.com/tim-kos/node-retry#retrytimeoutsoptions